jetbrains-core/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/telemetry/TelemetryHelper.kt

```diff
@@ -39,6 +42,10 @@ class TelemetryHelper(private val context: AmazonQAppInitContext, private val se
     private val responseStreamStartTime: MutableMap<String, Instant> = mutableMapOf()
     private val responseStreamTotalTime: MutableMap<String, Int> = mutableMapOf()
     private val responseStreamTimeForChunks: MutableMap<String, MutableList<Instant>> = mutableMapOf()
+    private fun getStartUrl(): String? {
```

Comment:
Instead of being a private function this can be a public function and can be reused across all the required files?

Reply:
It is private to keep the access minimal. If external access is necessary, by that time this should be public.

Reply:
We are anyway calling these lines multiple times in the previous files too, so this could probably be a util function?